### PR TITLE
Support debridge

### PR DESCRIPTION
**Pull request by mncdg**
Created new abstraction ChainSwapInfo that have 2 realization: SingleChainSwapInfo, CrossChainSwapInfo
Created new service DebridgeDlnProviderV1 that does requests:
- {TRADING_API}/v1.0/dln/order/create-order - for creating data for swap
- {TRADING_API}/v1.0/dln/order/quote - for estimation swap 
- {ORDER_API}/api/Orders/filteredList - finding orders
- {TRADING_API}/v1.0/dln/order/{orderId} - get order by id

New provider: debridge

Envs:
 - DEBRIDGE_TRADING_API
 - DEBRIDGE_ORDER_API

### CODE REVIEW

**Review by shestakoven — CHANGES_REQUESTED** [review]
Thanks for your PR, it's pretty good!

But there are few points:
- Let's move crosschain swaps to separate url path (for example /v1/crosschain) and add here `chain_id_to` and `chain_id_from` query params;

- After that I'm not sure that you need `ChainSwapInfo` anymore;

Anything else LGTM!

**Review by shestakoven — CHANGES_REQUESTED** [review]
This thing with allowance check looks critical. Correct me if I'm wrong. Anything else LGTM.

File: meta_aggregation_api/providers/debridge_dln_v1/debridge_dln_provider_v1.py [hunk 1]
```diff
@@ -153,7 +156,8 @@ async def get_swap_quote(
             'srcChainTokenInAmount': sell_amount,
             'dstChainId': chain_id_to,
             'dstChainTokenOut': buy_token,
-            'affiliateFeePercent': 0,
+            'affiliateFeePercent': int(buy_token_percentage_fee * 100),
+            "affiliateFeeRecipient": taker_address,
```
Comment (shestakoven) [inline meta_aggregation_api/providers/debridge_dln_v1/debridge_dln_provider_v1.py:160]: `'affiliateFeeRecipient': fee_recipient `

**Review by shestakoven — APPROVED** [review]
Looks to be working!

**Review by evahteev — APPROVED** [review]
LGTM

**Pull request merged**